client/lib/domains/resolve-domain-status.js

```diff
@@ -139,6 +139,17 @@ export function resolveDomainStatus(
 			};
 
 		case domainTypes.REGISTERED:
+			if ( domain.isPendingRenewal ) {
+				const pendingRenewalMessage = translate( 'Renewal in progress' );
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **11** times:
`translate( 'Domain renewal already in progress.' )` *ES Score: 7*